### PR TITLE
Fix example group issue with turnip

### PR DESCRIPTION
**Pull request by tevio**
* the previous fix for this - #19 - does not work with Rails 4.2.6, turnip 2.1.1, rspec 3.4.0 & knapsack latest, this fixes that fix

### CODE REVIEW

File: lib/knapsack/adapters/rspec_adapter.rb [hunk 1]
```diff
@@ -48,10 +48,8 @@ def bind_time_offset_warning
       end
 
       def self.test_path(example_group)
-        unless example_group[:turnip]
```
Comment (ArturT) [inline lib/knapsack/adapters/rspec_adapter.rb]: Would be good to still have backward compatibility with Turnip 1.x

You could check if Turnip exists and if the version is bellow 2.x then follow old approach.

```ruby
if defined?(Turnip) && Turnip::VERSION.to_i < 2
```

and we need test for both cases.

File: spec/knapsack/adapters/rspec_adapter_spec.rb [hunk 2]
```diff
@@ -100,10 +100,10 @@
     context 'with turnip features' do
       let(:current_example_metadata) do
         {
-          file_path: "./spec/features/logging_in.feature",
+          file_path: "gems/turnip-1.2.4/lib/turnip/rspec.rb",
```
Comment (ArturT) [inline spec/knapsack/adapters/rspec_adapter_spec.rb]: here is example with hard coded turnip version `1.2.4` it could be x.x.x

**Comment by tevio** [comment]
@ArturT I've added the extra case for Turnip <= 2, let me know if there is anything else needs doing on this PR :)

**Comment by ArturT** [comment]
@tevio PR looks good. Thank you! I'm going to merge it and release a new gem version by the end of this week.

**Comment by ArturT** [comment]
@tevio I've released version 1.12.2. Thanks for PR. :)

**Pull request merged**